### DISCUSSION TITLE
Web Cam View

### DISCUSSION THREAD

**Discussion by gene-git** [Motion-Project/motionplus#48] [General]
Hi - long time motion user (thank you!) , and now testing your new motionplus.

I set up 10 rtps netcams. When I go to the web view - it shows a grid of 6. If I click hambuger menu -> cameras -> All Cameras
the grid expands to 9. So last one is missing. If I click on the missing name in the left index it shows fine. There is plenty of room for a 4 x 4 grid in my browser it seems to me.

Is there a way (maybe I missed it) to to see all cameras, preferably, or at least a way to page the grid if larger?

thank you!

**Comment by Mr-Dave** [comment]
It is probably the browser max connection limitation.  Change the [stream_preview_method](https://motion-project.github.io/motionplus_config.html#stream_preview_method)

**Comment by gene-git** [comment]
Yep - that did it :)

thank you

**Comment by gene-git** [comment]
One thing I note - if I click on one of the steams - and then press 'back' button - instead of returning me to the overview it exits out of motionplus entirely - I can get back by clicking menu -> cameras -> all cameras -> "x" out of index list. 

Be good if there was a 1 click way to go back to 'all cam' view from 1 cam.

**Comment by gene-git** [comment]
Also while static does show all the cameras - they are no longer streaming - they are static - 

So I think question stands - how to get a grid with all cameras streaming.

I can work around this by reducing stream_preview_scale until it shows more cameras.

thanks

gene

**Comment by Mr-Dave** [comment]
The images will update on the main page.  It just won't be via a mjpg stream.  It is via a javascript update.

**Comment by gene-git** [comment]
Okay - would this be static images every few seconds or you expect it would it be fast enough to be seen a live stream?
For now I turned off static.

**Comment by Mr-Dave** [comment]
It is 2 seconds and coded into the html page.  You'd have to load/edit/create your own page is you want something different.  

[Default line](https://github.com/Motion-Project/motionplus/blob/9e61018a2e10175666cb9f45620968429b513de7/doc/samplepage.html#L868)

[Alternative webcontrol page request](https://github.com/Motion-Project/motionplus/issues/2)

**Comment by gene-git** [comment]
Understand thank you. I'm really liking MP so far and its working really well.  Maybe the 2secs could be a companion config variable for static case at some point? These are very minor things.

From all my testing so far it is mature enough to replace motion at this point and is quite a bit more efficient as well from what I see.

I use motion pass through and the netcam_high streams being saved nicely.
I do have another question for you  but not on the web page so will open new topic for that one.
